### PR TITLE
[meta] do not publish .github dir

### LINKED ISSUE #122: Add test folder to npmignore
The test folder occupies around 264K bytes. Is it necessary to include in the published package?

Related: #118.

### PR DESCRIPTION
**Pull request by david-fong**
Similar changes can be made to the dependencies of this package that you also maintain.

Fixes #122

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # gitignore
 node_modules
+.github
```
Comment (ljharb) [inline .npmignore:3]: this is fine, but negligible

File: .npmignore [hunk 2]
```diff
@@ -13,3 +14,4 @@ yarn.lock
 # dev scripts
 operations/*.js
 !operations/201*.js
+test/
```
Comment (ljharb) [inline .npmignore:17]: this is intentional - `npm explore foo && npm install && npm test` should work for all packages.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by ljharb — APPROVED** [review]
Seems pretty valueless, but now it doesn't hurt at least.

**Comment by ljharb** [comment]
@david-fong it's very unfortunate you deleted your fork :-/ that's an unrecoverable action that prevents me from making further changes or rebasing.

**Comment by david-fong** [comment]
@ljharb I'm super sorry I did that! I wasn't using my brain enough when I was deleting some of my forks :( I'll make sure not to delete forks with open PR's in the future. Again, sorry, and thank you so much for taking a look at this little PR. Happy new year!

**Comment by ljharb** [comment]
No worries, thanks for the contribution :-)

**Pull request merged**